packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2359,15 +2350,15 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
      dashboards:
        enabled: true
        label: grafana_dashboard
```

Comment:
So the labels will stay the same and does not need to match the rename?

Reply:
Yes, the label stays the same since it's given by `grafana.sidecar.dashboards.label`.

https://github.com/prometheus-community/helm-charts/blob/d3184e900b4120c85c782a1bfac2b62efe23fc0e/charts/kube-prometheus-stack/values.yaml#L532